### PR TITLE
run wmlindent and wmllint

### PR DESCRIPTION
**Pull request by cooljeanius**
just opening this as a draft for now; I might've broken some stuff in the process, so I'll want to get that sorted out before merging...

### CODE REVIEW

**Comment by cooljeanius** [comment]
ok it looks like I've run into wesnoth/wesnoth#1397 with this...

**Comment by cooljeanius** [comment]
actually maybe I should split this into separate PRs, one for `wmlindent`, and then another for `wmllint`...

**Comment by ForestDragon-wesnoth** [comment]
checking commit [234505e](https://github.com/ForestDragon-wesnoth/Spartan/pull/4/commits/234505e381858769ff4036105fdb225e755c00f6) not sure what changed in [hide_unit] in main

File: _main.cfg [hunk 1]
```diff
@@ -190,7 +190,7 @@ Third Note: it is recommended to play this scenario/campaign without eras or mod
...
... Hoplite,Merman Hunter,Merman Javelineer,Merman Netcaster,Merman Spearman,Merman Triton,Merman Warrior,Mudcrawler,Naga Dirkfang,Naga Fighter,Naga Guard,Naga High Guard,Naga Myrmidon,Naga Ophidian,Naga Ringcaster,Naga Shield Guard,Naga Sicarius,Naga Warrior,Naga Zephyr,Necromancer,Necrophage,Nightgaunt,Ogre,Orcish Archer,Orcish Assassin,Orcish Crossbowman,Orcish Grunt,Orcish Leader,Orcish Nightblade,Orcish Ruler,Orcish Slayer,Orcish Slurbow,Orcish Sovereign,Orcish Warlord,Orcish Warrior,Outlaw,Paladin,Peasant,Piglet,Pikeman,Pirate Galleon,Poacher,Ranger,Red Mage,Revenant,Roc,Rock Scorpion,Rogue,Royal Guard,Royal Warrior,Ruffian,Saurian Ambusher,Saurian Augur,Saurian Flanker,Saurian Oracle,Saurian Skirmisher,Saurian Soothsayer,Sea Serpent,Sergeant,Shadow,Shock Trooper,Silver Mage,Skeletal Dragon,Skeleton,Skeleton Archer,Sky Drake,Soulless,Spearman,Spectre,Swamp Lizard,Swordsman,Tentacle of the Deep,Thief,Thug,Transport Galleon,Trapper,Troll,Troll Hero,Troll Rocklobber,Troll Shaman,Troll Warrior,Troll Whelp,Vampire Bat,Walking Corpse,Water Serpent,White Horse,White Mage,Wild Wyvern,Wolf,Wolf Rider,Woodland Boar,Woodsman,Wose,Wose Sapling,Wose Shaman,Wraith,Wyvern Rider,Yeti,Young Ogre
```
Comment (cooljeanius) [inline _main.cfg]: can anyone tell what changed here? I can't...

File: macros/misc_events.cfg [hunk 2]
```diff
@@ -338,7 +338,7 @@ With the grave citizens, with the great throng.'"}
...
...dwan,Rihab,Riyad,Rizq,Ruhi,Rushd,Rushdi,Ruwayd,Saad,Saʿadah,Sab,Sabih,Sabir,Sabri,Saʿd,Saʿd al-Din,Sadad,Sadid,Sadiq,Saʿdun,Saʿid,Safi,Safiy,Safiy al-Din,Safuh,Safwah,Safwat,Safwan,Sahib,Sahir,Sahl,Saʾib,Saif,Saif al-Din,Sajid,Sajjad,Sakhr,Salah,Salah al-Din,Salamah,Salih,Salim,Salman,Sami,Samih,Samir,Samman,Saqr,Sariyah,Sati,Saud,Sayyid,Shaʿban,Shadi,Shadin,Shafi,Shafiq,Shahid,Shahin,Shahir,Shakib,Shakir,Shams al-Din,Shamal,Shamil,Shamim,Sharaf,Sharif,Shawqi,Shihab,Shihab al-Din,Shihad,Shuʿayb,Shukri,Shumayl,Siddiq,Sinan,Siraj,Siraj al-Din,Sofian,Subhi,Sufyan,Suhayb,Suhayl,Suhaym,Sulaiman,Sumrah,Suraqah,Suʿud,Tahir,Tahsin,Taym Allah,Taj,Taj al-Din,Talal,Talib,Tamim,Tamir,Tamam,Tammam,Taqiy,Tarif,Tariq,Taslim,Tawfiq,Tawhid,Taymullah,Taysir,Tayyib,Thabit,Thamir,Thaqib,Thawab,Thawban,ʿUbaidah,Ubaid,Ubayy,ʿUdayl,ʿUday,ʿUmar,Umarah,Umair,ʾaUrwah,Usaym,Usama,ʿUtbah,Uthal,Uthman,Waddah,Wadi,Wadid,Wafiq,Wahab,Wahhab,Wahid,Waʾil,Wajdi,Wajid,Wajih,Wakil,Walid,Walif,Waliy Allah,Waliy al-Din,Waqar,Waqqas,Ward,Wasif,Wasil,Wasim,Wazir,Yahya,Yaman,Yaʿqub,Yasar,Yasin,Yasir,Yazan,Yazid,Yunus,Yushua,Yusri,Yusuf,Zafar,Zafir,Zahid,Zahir,Zayd,Zaim,Zayn,Zarif,Zakarriya,Zaki,Zakwan,Ziyad,Zubayr,Zuhayr"}
```
Comment (cooljeanius) [inline macros/misc_events.cfg]: hm, I can't really tell what changed here, either...

File: units/races.cfg [hunk 3]
```diff
@@ -254,7 +254,7 @@
...
...n,Lempo,Lenthar,Lenvan,Leonard,Leraje,Lix Tetrax,Lor,Lorath,Loriel,Lorez,Lucifer,Lucifuge Rofocale,Maeglhir,Maeglhor,Magnus,Malal,Malphas,Malthus,Mammon,Marchocias,Maricha,Mastema,Melchom,Melchor,Mephisto,Mephistopholes,Mesphito,Merihim,Merloc,Moloch,Morax,Murdoc,Mulciber,Murmur,Naberius,Nale,Name,Nate,Nati,Nebiros,Neqa'el,Nergal,Nisroch,Noy,Noyga,Noyra,Noyze,Oeillet,Olivier,Optimus,Orcus,Orias,Ornias,Orobas,Ose,Oulotep,Pan,Paimon,Pazuzu,Penemue,Phlegathon,Phenex,Phobos,Pinem'e,Pithius,Piru,Pluto,Procell,Pruflas,Purson,Pytho,Python,Pythos,Rahab,Rahovart,Dune Riderel,Raum,Ravanal,Raven,Razakel,Regent,Regulus,Rimmon,Rhotomago,Ronove,Ronwe,Rosier,Sabnock,Sahaqel,Saleos,Samael,Samigina,Samyaza,Santa,Sariel,Satan,Satanachia,Satariel,Seir,Sephiroth,Sephit,Shaitan,Shamsiel,Shax,Shezmu,Sid,Sidragasum,Sin,Sith,Sitri,Sonnillon,Sorum,Soufflet,Spar,Stolas,Surgat,Tabaet,Taliz,Tamiel,Tan,Tannin,Tar,Taurus,Teeraal,Temeluchus,Thammuz,Theracar,Thor,Thorin,Thurin,Triel,Troy,Turiel,Turin,Turon,Umbriel,Urobach,Val,Valac,Valefar,Vapula,Vassago,Verrine,Vine,Vual,Xan,Xandros,Xaphan,Xen,Yael,Yanqui,Yeqon,Yeter'el,Yhangor,Yomiel,Yuki,Zagan,Zama,Zaqiel,Zepar,Zet,Zeth,Zhan,Zhangor,Zhunol,Zort,Zorg,Zudo,Zuno"
```
Comment (cooljeanius) [inline units/races.cfg]: another line where I can't really tell what changed...

File: macros/misc_events.cfg (hunk 2 above)
Comment (cooljeanius) [reply macros/misc_events.cfg]: oh wait it changed "Hakim" to "Dune Herbalist" because of the Dunefolk redesign; that's a mistake that I'll revert...

Comment (cooljeanius) [reply macros/misc_events.cfg]: (kinda like @knyghtmare always becoming "Dune Paragon"...)

File: _main.cfg (hunk 1 above)
Comment (cooljeanius) [reply _main.cfg]: ah, apparently it was "Giant Stoat" to "Frost Stoat", which is okay.

File: units/races.cfg (hunk 3 above)
Comment (cooljeanius) [reply units/races.cfg]: ah, it was another case of a Dunefolk name update...

**Comment by cooljeanius** [comment]
I opened wesnoth/wesnoth#7881 for the Dunefolk name changes part

**Comment by cooljeanius** [comment]
> I opened [wesnoth/wesnoth#7881](https://github.com/wesnoth/wesnoth/issues/7881) for the Dunefolk name changes part

...and wesnoth/wesnoth#7882 for the text size changes part

**Comment by ProditorMagnus** [comment]
changelog.cfg is invalid wml.

**Comment by cooljeanius** [comment]
> changelog.cfg is invalid wml.

I don't think it's even meant to be WML at all in the first place...

**Comment by cooljeanius** [comment]
ok I guess that's enough for this PR; further changes will be in a separate one

**Pull request merged**